danielMunteanu/src/main/java/danielCollectionTask/Order.java

```diff
@@ -0,0 +1,45 @@
+package danielCollectionTask;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static java.time.LocalDateTime.now;
+
+public class Order {
+    public long id;
+    int counter;
+    private LocalDateTime date;
+    public User user;
+    public List<Item> items;
+
+    public Order() {
+        this.date = now();
+        this.id = ++counter;
```

Comment:
in your case the id is type long and the counter is int -> Could you please align the data types?

Reply:
fixed,both long